contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/SpecialPermissionsList.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div data-test="special-permissions-list">
+    <div class="header-section">
+      <div class="title">
+        {{ specialPermissionsDetected$() }}
+      </div>
+      <div class="description">
+        {{ confirmDistributionRights$() }}
+      </div>
+    </div>
+
+    <div
+      v-if="isLoading"
+      class="loader-wrapper"
+    >
+      <KCircularLoader />
+    </div>
+
+    <template v-else-if="currentPagePermissions.length > 0">
+      <div class="permissions-box">
+        <KCheckbox
+          v-for="permission in currentPagePermissions"
+          :key="permission.id"
+          :checked="checkedIds.includes(permission.id)"
+          :label="permission.description"
+          class="permission-checkbox"
+          @change="togglePermission(permission.id)"
+        />
+      </div>
+
+      <div
+        v-if="totalPages > 1"
+        class="pagination"
+      >
+        <KButton
+          :disabled="currentPage === 1"
+          appearance="basic-link"
+          class="nav-button"
+          @click="previousPage"
+        >
+          &lt; Previous
+        </KButton>
+        <span class="page-indicator">{{ currentPage }} of {{ totalPages }}</span>
+        <KButton
+          :disabled="currentPage === totalPages"
+          appearance="basic-link"
+          class="nav-button"
+          @click="nextPage"
+        >
+          Next &gt;
+        </KButton>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+  import { computed, watch } from 'vue';
+  import { themePalette } from 'kolibri-design-system/lib/styles/theme';
+  import { useSpecialPermissions } from './composables/useSpecialPermissions';
+  import { communityChannelsStrings } from 'shared/strings/communityChannelsStrings';
+
+  const {
+    specialPermissionsDetected$,
+    confirmDistributionRights$,
+  } = communityChannelsStrings;
+
+  export default {
+    name: 'SpecialPermissionsList',
+    components: {
+    },
+    props: {
+      permissionIds: {
+        type: Array,
+        required: false,
+        default: () => [],
+      },
+      modelValue: {
+        type: Array,
+        required: false,
+        default: () => [],
+      },
+    },
+    emits: ['update:modelValue', 'update:allChecked'],
+    model: {
+      prop: 'modelValue',
+      event: 'update:modelValue',
+    },
+    setup(props, { emit }) {
+      const paletteTheme = themePalette();
+      const permissionIdsRef = computed(() => {
+        const ids = props.permissionIds || [];
+        return ids;
+      });
```

Comment:
Idem here, we don't need the computed since we already define a default value in the props definition.